.pre-commit-config.yaml

```
@@ -0,0 +1,64 @@
+default_language_version:
+  python: python3
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: "v5.0.0"
+    hooks:
+      - id: check-added-large-files
+        args: ["--maxkb=1024"]
+      - id: check-case-conflict
+      - id: check-docstring-first
+      - id: check-executables-have-shebangs
+      - id: check-json
+      - id: check-merge-conflict
+      - id: check-shebang-scripts-are-executable
+      - id: check-toml
+      - id: check-yaml
+        args: ["--allow-multiple-documents"]
+      - id: debug-statements
+      - id: detect-private-key
+      - id: end-of-file-fixer
+        exclude_types: [json, sql]
+      - id: file-contents-sorter
+        files: ^(requirements/\w*.txt)$
```

In this project there's only a single `requirements.txt` file
```suggestion
        files: ^requirements\.txt$
```

---

I updated the related commit, should be okay now.

---

I still see the old line

---

It seems this hook doesn't add value to this repo. The `requirements.txt` file has a special organization, which gets destroyed when package names are ordered alphabetically. I removed it.